src/libstore/build-result.hh

```diff
@@ -36,6 +36,7 @@ struct BuildResult
         NotDeterministic,
         ResolvesToAlreadyValid,
         NoSubstituters,
+        NoBuilders,
```

Comment:
```suggestion
        NoCompatibleBuilder,
```

Reply:
It may be worth adding special handling to Hydra for this build result (see https://github.com/NixOS/hydra/blob/250668a19fa4d8ff9a6176ee6c44ca3003adedf1/src/hydra-queue-runner/build-remote.cc#L360-L413) to get a nicer build status in the UI than "aborted".

Reply:
I made the change. Yeah, having Hydra support this in the future would be nice. It could say something like "no compatible build machines".